rest-api/src/main/resources/application.properties

```diff
@@ -2,3 +2,8 @@ spring.main.banner-mode=off
 spring.servlet.multipart.max-file-size=-1
 spring.servlet.multipart.max-request-size=-1
 security.user.password=not_used
+server.port=8443
+#security.require-ssl=true
```

Comment:
I assume we'll uncomment once we completely remove our http dependency correct?

Reply:
Yes, and this property is deprecated in current spring boot version we are using
so should use _server.ssl.enabled_